dmriprep/cli/run.py

```diff
@@ -301,26 +300,11 @@ def main():
     # Clean up master process before running workflow, which may create forks
     gc.collect()
 
-    # Sentry tracking
-    if not opts.notrack:
-        from ..utils.sentry import start_ping
-        start_ping(run_uuid, len(subject_list))
-
     errno = 1  # Default is error exit unless otherwise set
     try:
         dmriprep_wf.run(**plugin_settings)
     except Exception as e:
```

Comment:
The exception variable is not ever used : https://travis-ci.org/nipreps/dmriprep/jobs/636648631#L474

Reply:
Thanks. I believe I reinstated the logic in 85f0401